nf_core/pipeline-template/{{cookiecutter.name_noslash}}/.github/workflows/branch.yml

```diff
@@ -0,0 +1,15 @@
+name: nf-core branch protection
+# This workflow is triggered on PRs to master branch on the repository
```

Comment:
```suggestion
# This workflow is triggered on PRs to master branch on the repository
# It fails when someone tries to make a PR against the nf-core `master` branch instead of `dev`
```